### PR TITLE
refactor to allow any javascript

### PR DESCRIPTION
**Pull request by ghankerson**

### CODE REVIEW

File: src/atoms/CustomHtml/CustomHtml.test.js [hunk 1]
```diff
@@ -178,7 +83,172 @@ test('If the fallback src is not on our whitelist, and the script has no source,
...
+  expect(container.firstChild.innerHTML).toMatch(/<\/script>$/);
+  expect(container.querySelectorAll('script').length).toEqual(1);
+});
+
+test('Allows a google recaptcha', () => {
+  const doc = JSON.parse(recaptcha.body);
+  script.nodeData.attrs.html = doc.content[0].attrs.html;
+  script.nodeData.attrs.fallback_url =
```
Comment (etherbob) [inline src/atoms/CustomHtml/CustomHtml.test.js:93]: Is fallback_url doing something different than providing a linkable fallback here?

Comment (ghankerson) [reply src/atoms/CustomHtml/CustomHtml.test.js:93]: I am just changing the value between tests here

File: src/atoms/CustomHtml/CustomHtml.test.js [hunk 2]
```diff
@@ -162,10 +69,8 @@ test('If no fallback src is provided, and there is an unsourced script, place it
...
-  );
-  expect(container.querySelectorAll('iframe').length).toEqual(1);
+  expect(container.firstChild.innerHTML).toMatch(/<\/script>$/);
+  expect(container.querySelectorAll('script').length).toEqual(1);
 });
 
 test('If the fallback src is not on our whitelist, and the script has no source, all html is placed in iframe', () => {
```
Comment (etherbob) [inline src/atoms/CustomHtml/CustomHtml.test.js:76]: Is this still true?

Comment (ghankerson) [reply src/atoms/CustomHtml/CustomHtml.test.js:76]: I need to update the test descriptions - doing now

**Pull request merged**